.idea/dictionaries/Daniyar_Yeralin.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
```

Comment:
These `.idea` files seem to have slipped through - can you remove them from the PR? We tend not to include IDE-specific files in the project `.gitignore`, but for convenience you might find setting `.git/info/exclude` locally or a similar approach handy.